cmd/lambda/accounts/main.go

```diff
@@ -185,3 +245,44 @@ func newAWSSession() *session.Session {
 	}
 	return awsSession
 }
+
+// WriteAPIResponse - Writes the response out to the provided ResponseWriter
+func WriteAPIResponse(w http.ResponseWriter, status int, body interface{}) {
+	w.WriteHeader(status)
+	json.NewEncoder(w).Encode(body)
+}
+
+// ErrorHandler returns the appropriate HTTP code depending on the error recieved
+func ErrorHandler(w http.ResponseWriter, err error) {
+	var status int
+	var code string
+	var message string
+	log.Printf("Error: %+v\n", err)
+
+	// Determine status code
+	if gErrors.Is(err, errors.ErrNotFound) {
+		status = http.StatusNotFound
+		code = "NotFound"
+		message = err.Error()
+	} else if gErrors.Is(err, &errors.ErrValidation{}) {
+		status = http.StatusBadRequest
+		code = "RequestValidationError"
+		message = err.Error()
+	} else if gErrors.Is(err, errors.ErrConflict) {
+		status = http.StatusConflict
+		code = "Conflict"
+		message = err.Error()
+	} else {
+		status = http.StatusInternalServerError
+		code = "ServerError"
+		message = err.Error()
```

Comment:
We don't want to dump the Error fully in this scenario.  Could have sensitive data.

Reply:
I like the error dump. What sensitive data are we concerned about?

Reply:
This could have been any error bubbled up.  So I think the concern is database connection information, account information, etc.